content/en/news/_index.md

```diff
@@ -10,6 +10,10 @@ section1:
 
 section2:
   news:
+    - title: 'KubeSphere 3.0.0 GA: Born for Hybrid Cloud Apps'
+      description: That's a Killer! KubeSphere 3.0.0 is Now Generally Available!
+      image: 
```

Comment:
I assume [this image](https://ap3.qingstor.com/kubesphere-website/docs/20200830101950.png) can be added here.